packages/components/src/snackbar/README.md

```diff
@@ -36,17 +36,52 @@ const MySnackbarNotice = () => (
 );
 ```
 
-#### Props
+### Props
 
 The following props are used to control the display of the component.
 
--   `children`: (string) The displayed message of a notice. Also used as the spoken message for assistive technology, unless `spokenMessage` is provided as an alternative message.
--   `spokenMessage`: (string) Used to provide a custom spoken message in place of the `children` default.
--   `politeness`: (string) A politeness level for the notice's spoken message. Should be provided as one of the valid options for [an `aria-live` attribute value](https://www.w3.org/TR/wai-aria-1.1/#aria-live). Defaults to `"polite"`. Note that this value should be considered a suggestion; assistive technologies may override it based on internal heuristics.
-    -   A value of `'assertive'` is to be used for important, and usually time-sensitive, information. It will interrupt anything else the screen reader is announcing in that moment.
-    -   A value of `'polite'` is to be used for advisory information. It should not interrupt what the screen reader is announcing in that moment (the "speech queue") or interrupt the current task.
--   `onRemove`: function called when dismissing the notice.
--   `actions`: (array) an array of action objects. Each member object should contain a `label` and either a `url` link string or `onClick` callback function.
+#### `children`: `string`
+The displayed message of a notice. Also used as the spoken message for assistive technology, unless `spokenMessage` is provided as an alternative message.
+-   Required: Yes
+
+#### `spokenMessage`: `string`
+Used to provide a custom spoken message in place of the `children` default.
+-   Required: No
+-   Default: `children`
+
+#### `politeness`: `'polite' | 'assertive'`
+A politeness level for the notice's spoken message. Should be provided as one of the valid options for [an `aria-live` attribute value](https://www.w3.org/TR/wai-aria-1.1/#aria-live). Defaults to `"polite"`. Note that this value should be considered a suggestion; assistive technologies may override it based on internal heuristics.
+-   A value of `'assertive'` is to be used for important, and usually time-sensitive, information. It will interrupt anything else the screen reader is announcing in that moment.
+-   A value of `'polite'` is to be used for advisory information. It should not interrupt what the screen reader is announcing in that moment (the "speech queue") or interrupt the current task.
+-   Required: No
+-   Default: `'polite'`
+
+#### `actions`: `Action[]`
+An array of action objects. Each member object should contain a `label` and either a `url` link string or `onClick` callback function.
+-   Required: No
+-   Default: `[]`
```

Comment:
Some nitty formatting:

- let's add an empty line in between the prop's heading and the description paragraph
- let's add an empty line after the description paragraph and before the required/default part

```suggestion
#### `actions`: `Action[]`

An array of action objects. Each member object should contain a `label` and either a `url` link string or `onClick` callback function.

-   Required: No
-   Default: `[]`
```

Let's apply this formatting for all props

Reply:
Ah, good idea. Committed in https://github.com/WordPress/gutenberg/pull/45472/commits/44a53712146af88dc39fe17e895b0fbb8d7d7cd7.